vochain/indexer/process.go

```diff
@@ -24,7 +24,7 @@ import (
 var zeroBytes = []byte("")
 
 func nonNullBytes(p []byte) []byte {
-	if p == nil {
+	if len(p) == 0 {
```

Comment:
why? if the incoming byte slice is non-nil and length zero, then we don't need to replace it, as far as I can tell.

Reply:
Right, it was not necessary.